gulp/nodemon.js

```diff
@@ -4,15 +4,13 @@
   uses nodemon to run a server, watches for javascript and json changes
 */
 
-const fs = require('fs')
-const path = require('path')
-
+const config = require('./config.json')
 const gulp = require('gulp')
 const gutil = require('gulp-util')
+const fs = require('fs')
```

Comment:
Hi, sorry we didn't document this, but it would be good to keep dependencies grouped, like this: https://github.com/alphagov/govuk-prototype-kit/blob/master/server.js#L1-L22 and then alphabetical within those groups. Does that make sense?

Reply:
Okay, I get you.  So NPM first and then locals?

Reply:
yeh, Core first (things that come with Node, that you don't need to `npm install`, like fs.

Then NPM stuff

Then stuff in the Prototype Kit codebase itself (local)

Reply:
Copy that.